int32 nationality
int32 favourite_taste